common/src/main/java/org/apache/sedona/common/utils/GeometrySplitter.java

```diff
@@ -0,0 +1,321 @@
+package org.apache.sedona.common.utils;
```

Comment:
Please make sure all new files have Apache License 2.0 header

Reply:
That file did look a little funny at the top. Added it.